addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -83,6 +83,7 @@ def _domain_employee_id(self):
         help="For an Accrual Allocation, this field contains the theorical amount of time given to the employee, due to a previous start date, on the first run of the plan. This can be manually edited.")
     duration_display = fields.Char('Allocated (Days/Hours)', compute='_compute_duration_display',
         help="Field allowing to see the allocation duration in days or hours depending on the type_request_unit")
+    last_executed_carryover_date = fields.Date()
```

Comment:
Sorry I didn't think of it directly, but since the field isn't made to be displayed, we shouldn't have it translated, same for `actual_lastcall`:
```suggestion
    last_executed_carryover_date = fields.Date(export_string_translation=False)
```